packages/components/src/InputDate/InputDate.test.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import { cleanup, fireEvent, render } from "@testing-library/react";
+import { InputDate } from ".";
+
+afterEach(cleanup);
+
+it("renders a blank form by default", () => {
+  const { getByDisplayValue } = render(<InputDate onChange={jest.fn()} />);
+  expect(getByDisplayValue("")).toBeTruthy();
+});
```

Comment:
Probs add another test here to make sure the date picker isn't actually open

Reply:
https://github.com/GetJobber/atlantis/pull/783/commits/42f4d9a8bbd0e407ec9979c9ecd64907647ed958